devops/Dockerfile.base

```diff
@@ -32,6 +32,7 @@ RUN R -e 'install.packages("devtools"); library(devtools)'
 RUN R -e 'library(devtools); devtools::install_github("klutometis/roxygen")'
 RUN R -e 'BiocManager::install("impute", version = "3.9")'
 RUN R -e 'install.packages("missForest")'
+RUN R -e 'install.packages("https://cran.r-project.org/src/contrib/Archive/mnormt/mnormt_1.5-6.tar.gz", repos=NULL, type="source")'
```

Comment:
Do you need me to push this up to dockerhub?

Reply:
Yes, I just modified the base docker image to include the rest of the new dependencies for the opencpu server. Should make CI run faster since downloading those R packages takes a while.

Reply:
I had to change a few things on the base image to get it to build.  I added the commit to this PR and pushed the changes to dockerhub.